tests/test_archive.py

```diff
@@ -1,13 +1,24 @@
 import pytest
 import os
+import glob
 
 
-@pytest.fixture(params=os.listdir('tests/data/archives'))
-def archive_path(request):
+ARCHIVES_OPENING_DIR = 'tests/data/archives/opening_archives'
+ARCHIVES_NOT_OPENING_DIR = 'tests/data/archives/not_opening_archives'
+FILES_DIR = 'tests/data/files'
+
+
+@pytest.fixture(params=glob.glob(ARCHIVES_OPENING_DIR+'/*'))
+def opening_archive_path(request):
+    return request.param
+
+
+@pytest.fixture(params=glob.glob(ARCHIVES_NOT_OPENING_DIR+'/*'))
+def not_opening_archive_path(request):
     return request.param
 
 
-@pytest.fixture(params=os.listdir('tests/data/files'))
```

Comment:
I was wondering what could be the best between `os.listdir` and `glob`, and I found something about [os.scandir](https://docs.python.org/3/library/os.html#os.scandir), just to mention it. In our case, any of those solutions are good.

Reply:
Thanks, I'll check `scandir` in the future